src/components/dashboard/FaceVerification/hooks/useFaceTecSDK.js

```diff
@@ -23,9 +23,15 @@ const log = logger.child({ from: 'useFaceTecSDK' })
  * @return {void}
  */
 export default eventHandlers => {
-  // Configuration callbacks refs
+  // parse options
   const { onInitialized = noop, onError = noop } = eventHandlers || {}
-  const accessors = useRealtimeProps([onInitialized, onError])
+
+  // state vars
+  const [initialized, setInitialized] = useState(false)
+  const [lastError, setLastError] = useState(false)
```

Comment:
as we provining also an prop-style interface in addition to callback-style
let's also will return an error from the hook